platform/podjs/src/browserPod.ts

```diff
@@ -627,13 +820,18 @@ function determineNavBarColors(manifest: Manifest): { fg: string; bg: string } {
     return { fg, bg };
 }
 
+/**
+ * Create a new iframe with a title and a background color
+ * @param {string} title - The title of the page.
+ * @returns A promise that resolves to a DOM element.
+ */
 function createNavBarFrame(title: string): HTMLElement {
     const frame = document.createElement("iframe");
     frame.style.display = "block";
     frame.style.width = "100%";
     frame.style.height = "50px";
-    frame.frameBorder = "0";
-    frame.scrolling = "no";
+    frame.frameBorder = "0"; //TODO: @deprecated
```

Comment:
Does this work? `@deprecated` should show a notice in attributes or class names in an IDE, but in this case? It's not a class, or an argument. Also, what's deprecated? The value? The assignment?

Reply:
OK, I see. The IDE already marks that as deprecated. In that case, the comment is redundant (and also not in scope for this PR, I'd say)